fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -363,15 +364,28 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         StartupTimeline.onActivityCreateEndHome(this) // DO NOT MOVE ANYTHING BELOW HERE.
     }
 
+    private fun navigateToHomeOrJunoOnboarding() {
+        if (!settings().isJunoOnboardingShown) {
+            FxNimbus.features.junoOnboarding.recordExposure()
+            if (settings().junoOnboardingEnabled) {
```

Comment:
Should we have the`junoOnboardingEnabled` check first, else we record the exposure when a user is not enrolled in Juno Onboarding?

Reply:
Gotcha, thanks for the info 🙏

Reply:
Also, updated the top condition to only show juno onboarding when the user clicks on the app icon. In the super rare edge case that the new user makes firefox their default browser and opens the app through a url, then the usual behavior will occur, as opposed to showing juno onboarding and then forgetting the url user clicked. cc @rocketsroger